src/stdune/loc.ml

```diff
@@ -15,6 +15,8 @@ let in_file fn =
   ; stop = pos
   }
 
+let in_dir = in_file
```

Comment:
This alias is mostly for readability right now, but perhaps we'll give it better behavior down the road.